openrouteservice/src/main/java/heigit/ors/services/matrix/MatrixServiceSettings.java

```diff
@@ -39,6 +40,9 @@
 		value = AppConfig.Global().getServiceParameter("matrix", "maximum_locations");
 		if (value != null)
 			maximumLocations = Math.max(1, Integer.parseInt(value));
+		value = AppConfig.Global().getServiceParameter("matrix", "maximum_locations_flexible");
```

Comment:
I have added the parameter to the staging app.config, but it would be good to have it added to the app.config.sample file within the repository so that it is there ready for new instances